### ISSUE TITLE
Padding/margin

### ISSUE THREAD

**Issue by LoganDark** [sagb/alttab#104]
Maybe I want:

- The icon to be centered
- Some space around the text and icon
- Some space between the text and icon

This shouldn't be too hard, right?

**Comment by sagb** [comment]
Where do you draw the text if you center the icon?

**Comment by LoganDark** [comment]
> Where do you draw the text if you center the icon?

You center the icon horizontally.

**Comment by dxps** [comment]
@sagb Related to any possible padding, I just wanted to ask you how did you made the icons get centered in the 3rd screenshot?

This tool is great. Love using it in awesomewm btw.

**Comment by LoganDark** [comment]
> @sagb Related to any possible padding, I just wanted to ask you how did you made the icons get centered in the 3rd screenshot?
> 
> This tool is great. Love using it in awesomewm btw.

By making the icons wider. The image will be fit to the icon size without stretching, so you get that centered effect.

**Comment by dxps** [comment]
@LoganDark Thanks! I'll try that.

Meanwhile, I just discovered an ugly behavior: running in the existing session (using the same cmd flags as listed below) it all looks great, but having it started as part of the session using `.xsession` using:

```shell
❯ cat .xsession

alttab -bg "#002B36" -font xft:FiraSansCondensed -theme "Material-Black-Pistachio-Suru" -t 400x140 -p none &

❯ 
```

it looks like this:

![image](https://user-images.githubusercontent.com/14140226/113343005-e2a6d200-9337-11eb-873c-8a65cf2e3b55.png)

**Comment by sagb** [comment]
> > @sagb Related to any possible padding, I just wanted to ask you how did you made the icons get centered in the 3rd screenshot?
> > This tool is great. Love using it in awesomewm btw.
> 
> By making the icons wider via CLI flag. The image will be fit to the icon size without stretching, so you get that centered effect.

Yes. There is an exact command line at the botom of https://github.com/sagb/alttab/blob/master/doc/screenshots/screenshots.md

**Comment by sagb** [comment]
> I made it working fine by having it executed a little bit delayed by:

You can also try `alttab -w 1` without a delay.

**Comment by dxps** [comment]
@sagb  Thanks! I'll try that.

Now, the last issue I'm facing in one of the systems (both running `manjaro-awesome` edition with XFCE as DE) I'm using it. Although, there are multiple apps (and many of the non-minimized) open, `alttab` shows just two of them:

![image](https://user-images.githubusercontent.com/14140226/113506495-e6ac3d00-954d-11eb-8022-dca19e752dd3.png)

Any idea, please?

Thanks again!

**Comment by sagb** [comment]
Multiple desktops? Try `-d 1`

**Comment by dxps** [comment]
Thanks again for the feedback. Let me get back to you with the outcome.
In AwesomeWM there are actually multiple screens (I have 3 defined, although I'm just using the 1st one).

So, now I have:
- in `~/.config/awesome/autorun.sh`:
  ```shell
  run /home/dxps/apps/scripts/run_alttab.sh
  ```
- in `/home/dxps/apps/scripts/run_alttab.sh`:
  ```shell
  #!/bin/sh

  ## sleep 2
  alttab -w 1 -d 1 -bg "#002B36" -font xft:FiraSansCondensed -theme "Material-Black-Pistachio-Suru" -i 400x40 -t 400x140 -p none
  ```
After login, it's all good. Things might break later, if it is the case.
